app/posts/screening-invite/2025/09/2025-08-13-invite-breast-screening-pilot-user-research.md

```diff
@@ -0,0 +1,117 @@
+
+    Despite this, the key point is that most people still saw their message and attended their appointment, and many were ultimately     indifferent on what type of message they received. If people are indifferent, this could make it easier to change the type of digital channel we use to send messages.
+
+4) **Most of the sample didn’t follow a link to further guidance as they didn’t need to**
+Each message had a link (or a QR code for the letter) to a gov.uk publication page. Most people in this sample didn’t follow the link because they had gone through the process before and the key information was quickly available in the message.
+
+## Attending appointments and remembering where they are
+
+We’ve found out that:
+
+1)	**The change to digital messages can impact the people who use their letter as a reminder**
+ Two people said they forgot about their appointment because they didn’t have a letter, with one of them saying they need letters because they have memory loss issues.
+
+    Letters can have the advantage of being easily visible as you put them on fridges or other places at home. For some, messages on a phone can be harder to remember in comparison and we have previously seen there can be issues relocating an NHS App message in our pre-pilot user research.
+
+
+2)	**Letters do not need to act as reminders to people with strong calendar habits**
+Most of this sample added their appointment to either a physical or digital calendar. This suggests there could be a health inequality issue to consider here. 
+
+    Our fallback system could work for those who are able to set up reminders themselves, but not for people who are less able to do this due to things like learning difficulties or digital capability and would benefit from a letter instead of an SMS. 
+
+    From this sample, we cannot see the size of this potential issue due to a small sample size and it only containing people without 'special appointment' requirements.
+
+3)	**Late letters due to postal issues can also cause appointment problems**
+A participant had to reschedule their appointment due to not receiving their letter in time, despite the letter being sent 3 weeks in advance (over the Christmas period).
+
+## Getting to appointments without being sent a map
+
+Knowing the area or local landmarks can mean a map isn't needed for finding the main hospital or screening location.
+
+In Dorset the screening took place in Beales Department Store – a known location to almost the entire sample. The invitation and text reminder told people Beales was the location and most people reported they didn't need a map since they already knew where Beales was.
+
+It's also worth noting that if just the address was shared, it would have only said '64-68, Dolphin Centre’ in the first line, showing it could be worth investigating whether sharing only addresses is always the best way to direct people.
```

Comment:
"if just" – should that be "in cases where only"

also "provided" not "shared"